packages/ui/src/screens/account_details/components/balance/index.tsx

```diff
@@ -49,9 +51,20 @@ const Balance: FC<BalanceProps> = (props) => {
   const notEmpty = formatData.some((x) => x.value && Big(x.value).gt(0));
   const dataMemo = useShallowMemo(notEmpty ? formatData : [...formatData, empty]);
 
+  const [price, set] = useState<number>(0);
+  useEffect(() => {
+    const setPrice = async () => {
+      const { data } = await axios.get(
+        'https://api.coingecko.com/api/v3/simple/price?ids=coreum&vs_currencies=usd'
+      );
+      set(data.coreum.usd);
+    };
+    setPrice();
```

Comment:
Please add a catch to this, and handle gracefully.

Right now, it might break the app if axios throws an error and it's not catched.

Reply:
Got it I'll add the fix!

Reply:
Updated now with the fix! Made this try-catch fix on other file where the same call is being made as well.